### DISCUSSION TITLE
Repostitory structure for v0.15.1 is not compliant

### DISCUSSION THREAD

**Discussion by Emphantom4d** [hassio-addons/addon-tasmoadmin#171] [Q&A]
hassio-addons/addon-tasmoadmin
Repostitory structure for v0.15.1 is not compliant
:(

**Comment by frenck** [comment]
Is this a discussion? Question? or a bug report?
Nothing in your message rings a bell.

**Reply by Emphantom4d** [reply]
Hello. my English is bad.
yes, writes that Repostitory structure for v0.15.1 is not compliant

**Reply by frenck** [reply]
Could you please provide the steps you take to get that error?

**Reply by Emphantom4d** [reply]
Hacs > Integration > Custom Repositories > https://github.com/hassio-addons/addon-tasmoadmin.git    category-integration

**Reply by frenck** [reply]
HACS is not for add-ons...

https://hacs.xyz/docs/faq/addons/

**Reply by Emphantom4d** [reply]
thanks.
